app/templates/admin/sales/index.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <h1 class="ui header six wide column">{{t 'Tickets Summary'}}</h1>
+  </div>
+  <div class="row">
```

Comment:
Every row should have a column inside it with the content.  Please read up on Semantic UI's grid layout.